1103 have a seperate magazine section for the archive section jay

FireBoyAJ24 opened the pull request:
Added a separate magazine filter for the archive section. This makes it easy for the editors to add magazine filters but will require that all old articles are tagged using the magazine tag snippet.

FireBoyAJ24 committed:
Settig up the routable files

FireBoyAJ24 committed:
Re arranged the archive html and started implementing the routable pages

FireBoyAJ24 committed:
Section page is now working

FireBoyAJ24 committed:
Stable version 1. Routable pages are used with query string

FireBoyAJ24 committed:
Removed an unneccessary print

FireBoyAJ24 committed:
Added Video search capability V1 with filters

FireBoyAJ24 committed:
Adding a link to the Issues Page

FireBoyAJ24 committed:
num of videos per page is different + fixed link query string for all section

FireBoyAJ24 committed:
Solved the issue when the video_section is not defined in the section archive section

FireBoyAJ24 committed:
Changed thlabel of the link

FireBoyAJ24 committed:
Added a Magazine Tag to the article page and the archive page

FireBoyAJ24 committed:
Merge the new archive page with he magazine section filters

FireBoyAJ24 committed:
Work on getting to display the magazine filter

FireBoyAJ24 committed:
Finished add the magazine filter to the html and css

FireBoyAJ24 committed:
Changed where the magazine snippet panel is present on the article page settings

FireBoyAJ24 committed:
Cleaned up the code

FireBoyAJ24 committed:
Added hardcoded rules to remove the magazine and the marketspace from the section filters

FireBoyAJ24 committed:
Temporary solution for adding Spoofs, Sex Issue and

SamuelmdLow reviewed (changes requested):
`ArchivePage` content panels does not include the default `Page` content panels. This means that you can't set default Page fields like title. If you delete your old archivepage you won't be able to create a new one because required `Page` fields are impossible to fill in admin. 

To fix this you should make it `content_panels = Page.content_panels +[...` so that the default `Page` fields can be filled

FireBoyAJ24 committed:
Added the page’s content panels to the content panels of the archive page

FireBoyAJ24 committed:
Adjusted the sentence of one of the help panel

SamuelmdLow commented on archive/templates/archive/objects/archive_filter.html:
```diff
@@ -0,0 +1,187 @@
+                <a href="{% pageurl self %}{{ request|query_string }}">All sections</a>
+            </li>
+            {% for section in sections %}
+                {% if section.title != "The Ubyssey Student Marketplace" and "20" not in section.title and  section.title != "Pages" %}
```
This seems like it will easily break. Perhaps instead we should add a checkbox field in sectionPage that, when checked, excludes the section from the Archive

SamuelmdLow reviewed (changes requested):
This works. I do think it would be better to rely on the categoriesSnippet instead of new models though. The website already has the magazine articles organized into categories so we could avoid having to relabel years worth of magazine articles if we added the Magazine CategorySnippet as a filter instead.

FireBoyAJ24 committed:
Removed the magazine Snippets

FireBoyAJ24 committed:
Redesign of the magazine and spoof filters

FireBoyAJ24 committed:
Fixed the mobile dropdown scrolling problem

FireBoyAJ24 committed:
Cleaned up the migrations

FireBoyAJ24 committed:
Fixed some bugs on the archive filters

FireBoyAJ24 commented:
I have incorporated Sam's ideas. Cleaned up the CSS and HTML which includes the current issues that the archive page is facing.

There will be a minimum of 4 category snippets needed.

- 1 for the self

- 3 for the spoofs

In terms of adding the category to articles... It is incredibly simple. We just need to add the above snippets to the sections (For the spoofs it is simple but for the self I had to add a category selection place for this to work)
Up until now, we should not be touching any articles. Effort up to now **MEDIUM**
However, now we have some fun.
From what I have found, the old magazine articles are found in the magazine section page. The only issue is that we will have to manually label each and every (Live and public) article to a category snippet.
The number of snippets here can be 2~3. The effort, **HARD (compromises) - IMPOSSIBLE (no compromises).**

FireBoyAJ24 committed:
Provide further control for the magazine and spoof filters

FireBoyAJ24 committed:
Fixed more archive filter and archive request bugs

SamuelmdLow committed:
fixed spoof sidebar active class
